pod/video_search/management/commands/create_pod_index.py

```diff
@@ -19,6 +16,10 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         """Create the Elasticsearch Pod index."""
-        delete_index_es()
+        try:
+            delete_index_es()
+            self.stdout.write(self.style.WARNING("The Pod index has been deleted.."))
+        except exceptions.NotFoundError:
+            self.stdout.write(self.style.WARNING("Pod index not found."))
```

Comment:
Je rajouterai "Pod index not found **on ElasticSearch server**."

Reply:
fait